### PR TITLE
Fix typo of MD5-LOW to SHA-LOW

### PR DESCRIPTION
**Pull request by crutchcorn**
I noticed while reading through the specification that there was a typo in the UUIDv5 table indicating an MD5-LOW encoding, when it should be SHA-LOW instead.

This PR:
- Fixes this issue
- Updates `sha` to `sha1`
- Fixes another mention of MD5 in UUIDv5

### CODE REVIEW

**Comment by cabo** [comment]
Also fix

```
For more information on MD5 security considerations see {{RFC6194}}.
```

File: docs/draft-00/draft-ietf-uuidrev-rfc4122bis.html [hunk 1]
```diff
@@ -1267,7 +1267,7 @@ <h3 id="name-uuid-version-5">
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |var|                        sha_low                            |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                            md5_low                            |
+|                            sha_low                            |
```
Comment (mcr) [inline docs/draft-00/draft-ietf-uuidrev-rfc4122bis.html:1270]:
```suggestion
|                            sha1_low                           |
```

File: docs/draft-00/draft-ietf-uuidrev-rfc4122bis.html [hunk 2]
```diff
@@ -1267,7 +1267,7 @@ <h3 id="name-uuid-version-5">
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |var|                        sha_low                            |
```
Comment (mcr) [inline docs/draft-00/draft-ietf-uuidrev-rfc4122bis.html:1268]:
```suggestion
|var|                        sha1_low                           |
```

**Comment by cabo** [comment]
Why are you editing the HTML?

**Comment by kyzer-davis** [comment]
> Why are you editing the HTML?

Addressing it, I will replicate these to the markdown file on this branch.

**Comment by crutchcorn** [comment]
Apologies for the confusion regarding HTML/MD edits. This is my first contribution to any RFC ever, so I'm admittedly a bit lost 😅

I'll correct for future contributions

**Pull request merged**